cart.spec

```diff
@@ -0,0 +1,82 @@
+Name:          cart
+Version:       0.0.1
+Release:       1%{?dist}
+Summary:       CaRT
+
+License:       Apache
+URL:           https//github.com/daos-stack/cart
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: libfabric-devel
+BuildRequires: pmix-devel
+BuildRequires: openpa-devel
+BuildRequires: mercury-devel
+BuildRequires: ompi-devel
+BuildRequires: libevent-devel
+BuildRequires: boost-devel
+BuildRequires: libuuid-devel
+BuildRequires: hwloc-devel
+BuildRequires: openssl-devel
+BuildRequires: libcmocka-devel
+BuildRequires: libyaml-devel
+#Requires:
+
+%description
+
+Collective and RPC Transport (CaRT)
+
+CaRT is an open-source RPC transport layer for Big Data and Exascale
+HPC. It supports both traditional P2P RPC delivering and collective RPC
+which invokes the RPC at a group of target servers with a scalable
+tree-based message propagating.
+
+%prep
+%setup -q
+%setup -q -a 1
+
+
+%build
+# remove rpathing from the build
+find . -name SConscript | xargs sed -i -e '/AppendUnique(RPATH=.*)/d'
```

Comment:
This is very ugly, far preferable would be to have a --no-rpath option to scons, which would not be difficult to add.

Reply:
Fair enough.  But I'm on a deadline to get RPMs for the whole DAOS stack, so I'd like to defer this for future work.